Makefile

```diff
@@ -65,7 +65,7 @@ SYSTEM_TEST_TARGET?=./tests/system
 NOSETESTS_OPTIONS?=--process-timeout=90 --with-timer -v --with-xunit --xunit-file=build/TEST-system.xml
 
 .PHONY: check-full
-check-full: update check golint staticcheck
+check-full: update check staticcheck
```

Comment:
I'm not super keen on removing targets from check-full -- it's no longer "full" then, adding another thing for developers to remember to run.

Can we either run golint twice (probably no big deal), or explicitly list the steps to run in CI in the non-lint stage?